.travis.yml

```diff
@@ -7,14 +7,11 @@ node_js:
   - "6"
   - "7"
 
-matrix:
-  allow_failures:
-    - node_js: "0.12"
-    - node_js: "node"
- 
 branches:
-  except:
-    - canary
+  only:
+    - master
+    - beta
+    - release
```

Comment:
Yes, it will build PRs but not branch builds when pushing to origin. This avoids running tests twice for the same PR (Travis/push and Travis/pr).